### DISCUSSION TITLE
Reference previous AI answer?

### DISCUSSION THREAD

**Discussion by sergey-icvm** [openai-php/client#368] [Q&A]
Hi, maybe it is a dumb question, but I didn't find it in the docs or previous answers here.
I am building a tool for automating some prompts.  But how to make the AI remember previous answers?
For example, a piece of code looks like this:
```
$result = $client->chat()->create([
	'model' => 'gpt-3.5-turbo',
	'messages' => [
			['role' => 'user', 'content' => $_POST['prompt']],
	],
]);
```
So if I make calls like that, does it create a new chat every time?
If that is so, how do I connect to the previous answer, so I can ask the AI something like this: "Use your answer [r3] to make [new task here].
When I try something like that, it doesn't see what it said to me before.

**Comment by hschimpf** [comment]
Take a look at the [docs](https://platform.openai.com/docs/guides/text-generation/chat-completions-api)

> The user messages provide requests or comments for the assistant to respond to. Assistant messages store previous assistant responses, ...

The API is stateless, so you are in charge of passing the conversation history to the AI.
